app/views/wikis/index.html.erb

```diff
@@ -10,14 +10,12 @@
     <th>Wiki(click link to edit) </th>
     <th>Wiki(click link to delete) </th>
   </tr>
-<% @user.wikis.each do |wiki| %>
-  <tr>
-    <td><%= link_to wiki.title, edit_user_wiki_path(@user,wiki), class: 'btn btn-success' %></td>
-    <td><%= link_to "delete", [wiki], method: :delete %></td>
-  </tr>
-<% end %>
-  </div>
-</div>
+  <% @wikis.each do |wiki| %>
+    <tr>
+      <td><%= link_to wiki.title, edit_wiki_path(wiki), class: 'btn btn-success' %></td>
```

Comment:
To make use of Pundit here in the view template, use `policy(...).edit?`. This checks if the user is authorised to edit the wiki and only shows them the edit link if they are authorised:

```diff
- <td><%= link_to wiki.title, edit_wiki_path(wiki), class: 'btn btn-success' %></td>
+ <% if policy(wiki).edit? %>
+   <td><%= link_to wiki.title, edit_wiki_path(wiki), class: 'btn btn-success' %></td>
+ <% end >%
```